oxd-server/src/main/java/org/gluu/oxd/server/persistence/SqlPersistenceServiceImpl.java

```diff
@@ -43,8 +43,8 @@ private void createSchema() {
 
             Statement stmt = conn.createStatement();
 
-            stmt.addBatch("create table if not exists rp(id varchar(36) primary key, data varchar(65534))");
-            stmt.addBatch("create table if not exists expired_objects( key varchar(50), value varchar(65534), type varchar(20), iat TIMESTAMP, exp TIMESTAMP)");
+            stmt.addBatch("create table if not exists rp(id varchar(36) primary key, data varchar(50000))");
+            stmt.addBatch("create table if not exists expired_objects( obj_key varchar(50), value varchar(50000), type varchar(20), iat TIMESTAMP NULL DEFAULT NULL, exp TIMESTAMP NULL DEFAULT NULL)");
```

Comment:
`key` -> `obj_key` is incompatible change, lets avoid it.

Reply:
Mysql is not accepting `key` as a column name.